Sasquatch/Sasquatch.xcodeproj/project.pbxproj

```diff
@@ -1139,7 +1149,7 @@
 		B2C97B5A21064A5300042A03 /* SasquatchPuppet */ = {
 			isa = PBXGroup;
 			children = (
-				B2C97B8A2106535500042A03 /* Assets.xcassets */,
+				040856182231C85600977E8F /* Supporting Files */,
```

Comment:
What happened to assets? Not used?

Reply:
I moved that to *Supporting Files* group as same as other folder structure.